src/js/apps/forms/form/form_app.js

```diff
@@ -155,7 +155,7 @@ export default App.extend({
   onFormServiceError(errors) {
     if (this.loadingModal) this.loadingModal.destroy();
 
-    if (errors[0].status === 403) {
+    if (errors[0].status === '403') {
```

Comment:
Seems like it might be better to allow both, casting `status` to an integer:

```suggestion
    const status = parseInt(errors[0].status);

    if (status === '403') {
```

Reply:
That's been updated to use `parseInt()`.